### ISSUE TITLE
a11y: 'Download' button is not accessible via keyboard

### ISSUE THREAD

**Issue by nwpappas** [swagger-api/swagger-ui#8881]
### Q&A (please complete the following information)
 - OS: all
 - Browser: all

### Describe the bug you're encountering
 'Download' button under 'Try it now' results is not accessible via keyboard.

### To reproduce...

Using the keyboard...
1. Navigate to https://petstore.swagger.io/#/user/createUsersWithArrayInput
2. Navigate to and active 'Try it out' button
3. Navigate to and active 'Execute' button
4. Navigate to 'Download' button
5. Verify that 'Download' is accessible via keyboard

### Expected behavior
 'Download' button should be accessible via keyboard.

### Screenshots
![Screenshot 2023-06-02 at 10 57 17 AM](https://github.com/swagger-api/swagger-ui/assets/2529208/79920499-df7a-4b9c-b673-3a1856950a1f)

### Additional context or thoughts
'Download' button is not accessible via keyboard, which prevents the users relying on the keyboard to perform their fundamental task.

**Comment by char0n** [comment]
Addressed in https://github.com/swagger-api/swagger-ui/pull/8974
